main.go

```diff
@@ -65,5 +65,5 @@ func main() {
 		console.Fatalln("Go runtime version check failed.", err)
 	}
 
-	minio.Main(os.Args, os.Exit)
+	minio.Main(os.Args)
```

Comment:
I couldn't able to get `os.Exit(0)` and related tests.  Are you talking about functional tests?